frontend/server/src/ProblemDeployer.php

```diff
@@ -10,7 +10,7 @@ class ProblemDeployer {
     const UPDATE_CASES = 1;
     const UPDATE_STATEMENTS = 2;
     const CREATE = 3;
-
+    const ZIP_MAX_SIZE = '100000000';
```

Comment:
puede esto ser un entero? hay que intentar evitar convertir innecesariamente las cosas a cadenas.

```suggestion
    const ZIP_MAX_SIZE = 100 * 1024 * 1024;  // 100 MiB
```